PROPOSAL.md

```diff
@@ -12,13 +12,13 @@ The primary interest of this proposal is to detail the contract interface itself
 
 This proposal consists of two contracts, a factory “deployer” contract and the actual smart wallet interface.
 
-# Contract 1: The Factory
```

Comment:
What's the reason behind changing the header depth? I think I prefer starting things at H1